### ISSUE TITLE
Not able to build J2ObjC because of ABSL lib is not configured in the project

### ISSUE THREAD

**Issue by sarsonj** [google/j2objc#2403]
Hi,

I just tried to build latest version of J2ObjC and it looks like, that new version of protoc compiler or/and related tools now depends on the ABSL lib. But that lib is separate lib, not part of protobuf lib (at least when installed using Homebrew). 

The pull request #2402 fixes this - new env ABSL_ROOT_DIR should point to where ABSL is installed.

**Comment by tomball** [comment]
I'm sorry you went to so much work: our external "how to build j2objc" doc
was missing the abseil installation step that's in our internal doc (can't
share the two, as they're on different systems). I gave a full answer
regarding this in your pull request; the TL;DR is to run "brew install
abseil". I appreciate your contribution, but think this is a simpler
solution.

**Comment by sarsonj** [comment]
Attaching screenshot, where both libs (protobuf and abseil) were installed using Homebrew. The Abseil is installed into separate folder (by default).
![Image](https://github.com/user-attachments/assets/a60b9ebd-fbba-402a-9a5d-693d27590b2f)

**Comment by tomball** [comment]
I found this by asking “how is this working on internal machines but not
public”, compared the two setup docs’ protobuf sections, and found the
abseil install command was added to the inner setup doc maybe six months
ago. Since then, I finally got an Apple Silicon laptop upgrade, so I had a
clean system where I followed the install doc step-by-step.

What we haven't made clear is what PROTOBUF_ROOT_DIR's purpose is. We don't
actually care what about the root dir, we instead need the include and
library paths using it:

   - PROTOBUF_INCLUDE_PATH: what the include path for the protobuf C++
   headers is.
   - PROTOBUF_LIB_PATH: where the protobuf static library is for linking.

PROTOBUF_ROOT_DIR is your Homebrew root directory, so if your Homebrew
directory is /usr/local/homebrew (the default), then the include path
becomes /usr/local/homebrew/include, and the library path is
/usr/local/homebrew/lib. Note there's no Homebrew cask path, since
installation creates symlinks from the cask's headers to homebrew/include,
and from the cask's libraries to homebrew/lib. So when Abseil is installed,
homebrew/include has both protobuf and abseil headers, and so on for
libraries.
